constraints.txt

```diff
@@ -413,7 +413,7 @@ jsonschema==4.21.1
 junit-xml==1.9
 jupyter_client==8.6.0
 jupyter_core==5.7.1
-keyring==24.3.0
+keyring==24.3.1
```

Comment:
@mikaeld `bigeye-airflow` requires `keyring>=24.3.1`. Is it ok to bump the version here?

Reply:
It's preferable to not modify the constraints file. However, you can add dependency overrides in a separate `requirements-override.txt` (or similarly named file) and install them after the existing pip install step in the docker image and in the local environment. [I have done something similar in the past](https://github.com/mozilla/telemetry-airflow/pull/1792/files), except I specified the requirements to override directly in the Dockerfile because it was a very temporary fix for an upstream issue.

Reply:
I added a `requirements-override.txt`. Let me know if the setup looks reasonable